extensions/ql-vscode/src/queryserver-client.ts

```diff
@@ -171,6 +171,16 @@ export class QueryServerClient extends DisposableObject {
       args.push('--old-eval-stats');
     }
 
+    if (await this.cliServer.cliConstraints.supportsStructuredEvalLog()) {
+      args.push('--evaluator-log');
+      args.push(`${this.opts.contextStoragePath}/structured-evaluator-log.json`);
```

Comment:
This is fine for now. And we will make sure this writes to the query storage location later.